run.sh

```diff
@@ -3,6 +3,7 @@
 while true
 do
     git pull
+    "${1:-python}" -m poetry update
     "${1:-python}" -m poetry install
```

Comment:
I don't think we need install, since there's update.

Reply:
We should probably leave it in unless someone tests it and confirms one way or the other.